frontend-react/cypress/support/commands/api.js

```diff
@@ -0,0 +1,61 @@
+import { GLOBAL_STORAGE_KEYS } from "../../../src/utils/SessionStorageTools";
+
+// NOTE: all of these commands expect cy.login() to have been called beforehand
+const accessToken = window.localStorage.getItem(
+    GLOBAL_STORAGE_KEYS.OKTA_ACCESS_TOKEN
+);
+
+const requestOptions = {
+    bearer: `Bearer ${accessToken}`,
+    timeout: 2 * 60 * 1000, // TODO: is two minutes enough?
```

Comment:
That's a great question, haha! For `production`, typically things not loading within 2 mins will eventually timeout by 3 mins. I have not seen fetches with a timeout larger than 3 mins.

Reply:
Oooof a three-minute fetch... 😆😭 

I'll increase this by one minute, and we'll see if we need to adjust as we play around with these specs against staging/production in the future.